### PR TITLE
Update interfaces/webrtc-identity.idl

### PR DESCRIPTION
**Pull request by stephenmcgruer**
Manual version of https://github.com/web-platform-tests/wpt/pull/20662 with WebIDL dependency added

### CODE REVIEW

**Comment by stephenmcgruer** [comment]
@foolip PTAL. Comparing this to the results in https://github.com/web-platform-tests/wpt/pull/20662, looks much saner to me; positive increases in both numerator + denominator on all three browsers.

**Comment by stephenmcgruer** [comment]
@foolip PTAL. Comparing this to the results in https://github.com/web-platform-tests/wpt/pull/20662, looks much saner to me; positive increases in both numerator + denominator on all three browsers.

File: interfaces/webrtc-identity.idl [hunk 1]
```diff
@@ -62,6 +66,29 @@ interface RTCIdentityAssertion {
     attribute DOMString name;
 };
 
+partial interface RTCError {
+  readonly attribute long? httpRequestStatusCode;
```
Comment (foolip) [inline interfaces/webrtc-identity.idl]: The "Partial interface RTCError: member names are unique" subtest fails, and indeed this is already in https://w3c.github.io/webrtc-pc/#rtcerror-interface.

**Review by foolip — CHANGES_REQUESTED** [review]
@stephenmcgruer can you find the spec change and report the duplication problem here? If a webrtc-pc update is in flight we might have to update both together here :/

File: interfaces/webrtc-identity.idl (hunk 1 above)
Comment (jan-ivar) [reply interfaces/webrtc-identity.idl]: Looks like https://github.com/w3c/webrtc-pc/pull/2390 missed parts of `httpRequestStatusCode` in the move. I'll file a PR.

**Comment by stephenmcgruer** [comment]
The RTCError duplication problem should be fixed now, PTAL @foolip

File: interfaces/webrtc-identity.idl (hunk 1 above)
Comment (foolip) [reply interfaces/webrtc-identity.idl]: That was https://github.com/w3c/webrtc-pc/pull/2428. Thanks!

**Review by foolip — APPROVED** [review]

**Pull request merged**